kbase/narrative
# Dataup 224 add check boxes to staging

# Description of PR purpose/changes
![Screen Shot 2020-10-16 at 6 19 10 PM](https://user-images.githubusercontent.com/4184019/96325390-3321b180-0fdc-11eb-8605-1f729a1d2ae1.png)

Finally ready in all of its glory - multi-select checkboxes. Phew! It's been a wild ride - thanks for coming along y'all

Requirements: 
* Make file type selection box red for each uploaded file when a file type has NOT been selected. (Without an autodetect implemented, all files will show a red file type selector by default). The checkbox for that row is also default disabled.

* Until any file type is selected the check all checkbox is disabled 

* When a file type is selected for that row, the red box color goes back to the original (grey) color and checkboxes are enabled.

* Import Selected button is disabled until a file is selected. 

* On selection of file(s) and clicking import selected new importers will be kicked off. One for each file. 

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-224

# Testing Instructions
- [x] Tests pass locally, to isolate run stagingAreaViewer-spec.js 
- [x] Changes available by spinning up a local narrative:
- Navigate to the data import tab
- Make sure you have at least one file added
- Before selecting data type the dropdown should be red placeholder text "Select a type"
- On clicking in the border and text color should change to black
- If you click away without selecting a type text color should return to placeholder state 
- If you select a type the text and border should stay black
- Checkboxes should be enabled
- On checking a checkbox the import selected button should become enabled 

# Dev Checklist:
